presto-raptorx/src/main/java/com/facebook/presto/raptorx/storage/OrcFileRewriter.java

```diff
@@ -0,0 +1,194 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.raptorx.storage;
+
+import com.facebook.presto.orc.OrcDataSource;
+import com.facebook.presto.orc.OrcReader;
+import com.facebook.presto.orc.OrcRecordReader;
+import com.facebook.presto.orc.OrcWriter;
+import com.facebook.presto.orc.OrcWriterStats;
+import com.facebook.presto.spi.Page;
+import com.facebook.presto.spi.PrestoException;
+import com.facebook.presto.spi.block.Block;
+import com.facebook.presto.spi.block.DictionaryBlock;
+import com.facebook.presto.spi.type.Type;
+import com.facebook.presto.spi.type.TypeManager;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import io.airlift.log.Logger;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InterruptedIOException;
+import java.util.BitSet;
+import java.util.List;
+import java.util.Map;
+
+import static com.facebook.presto.raptorx.RaptorErrorCode.RAPTOR_STORAGE_ERROR;
+import static com.facebook.presto.raptorx.util.OrcUtil.createOrcFileWriter;
+import static com.facebook.presto.raptorx.util.OrcUtil.createOrcReader;
+import static com.facebook.presto.raptorx.util.OrcUtil.createOrcRecordReader;
+import static com.facebook.presto.raptorx.util.OrcUtil.fileOrcDataSource;
+import static com.google.common.collect.ImmutableList.toImmutableList;
+import static io.airlift.units.Duration.nanosSince;
+import static java.lang.Long.parseLong;
+import static java.lang.Math.toIntExact;
+
+public final class OrcFileRewriter
```

Comment:
"highker on Feb 4  Member
OrcFileRewriter sounds like a writer rather than a util. Maybe OrcFileRewrite or RewriteUtil?"

OrcFileWriter and OrcFileRewriter both uses OrcUtil.  Seems exact what it means. I will keep it.